engine/class_modules/sc_death_knight.cpp

```diff
@@ -7710,15 +7710,16 @@ void death_knight_t::default_apl_frost()
   cooldowns -> add_action( "fireblood,if=buff.pillar_of_frost.remains<=8&buff.empower_rune_weapon.up" );
 
   // Pillar of Frost
-  cooldowns -> add_action( this, "Pillar of Frost", "if=cooldown.empower_rune_weapon.remains", "Frost cooldowns" );
+  cooldowns -> add_action( this, "Pillar of Frost", "if=cooldown.empower_rune_weapon.remains|!talent.breath_of_sindragosa.enabled", "Frost cooldowns" );
```

Comment:
If ERW is used on cooldown (cf. the L7717 addition), isn't it better to keep trusting that ERW is on cd before pressing PoF? It should be better to lose an ERW gcd to cast PoF than a PoF gcd to cast ERW, shouldn't it?

Reply:
While id be inclined to agree, the sims tend to disagree, not to much of an extent, but there is a difference. 
https://www.raidbots.com/simbot/report/ifDNmF9sGzZDrRQtsHpToQ